src/openssl/lib_ssl.cr

```diff
@@ -38,9 +45,53 @@ lib LibSSL
   fun ssl_shutdown = SSL_shutdown(handle : SSL) : Int
 
   fun ssl_free = SSL_free(handle : SSL)
+  fun ssl_ctx_set_cipher_list = SSL_CTX_set_cipher_list(ctx : SSLContext, ciphers : Char*) : Int
   fun ssl_ctx_use_certificate_chain_file = SSL_CTX_use_certificate_chain_file(ctx : SSLContext, file : UInt8*) : Int
   fun ssl_ctx_use_privatekey_file = SSL_CTX_use_PrivateKey_file(ctx : SSLContext, file : UInt8*, filetype : SSLFileType) : Int
   fun ssl_set_bio = SSL_set_bio(handle : SSL, rbio : LibCrypto::Bio*, wbio : LibCrypto::Bio*)
+
+  #SSL_CTRL_SET_TMP_RSA = 2
+  #SSL_CTRL_SET_TMP_DH = 3
+  SSL_CTRL_SET_TMP_ECDH = 4
+
+  SSL_CTRL_OPTIONS = 32
+  SSL_CTRL_MODE = 33
+  SSL_CTRL_CLEAR_OPTIONS = 77
+  SSL_CTRL_CLEAR_MODE = 78
+
+  SSL_OP_ALL = 0x80000bff_u32
+  SSL_OP_DONT_INSERT_EMPTY_FRAGMENTS = 0x00000800_u32
+  SSL_OP_NO_COMPRESSION = 0x00020000_u32
+  SSL_OP_NO_SSLv2 = 0x01000000_u32
+  SSL_OP_NO_SSLv3 = 0x02000000_u32
+  SSL_OP_NO_TLSv1 = 0x04000000_u32
+  SSL_OP_NO_TLSv1_1 = 0x10000000_u32
+  SSL_OP_NO_TLSv1_2 = 0x08000000_u32
+  SSL_OP_NO_SESSION_RESUMPTION_ON_RENEGOTIATION = 0x00010000_u32
+  SSL_OP_SINGLE_ECDH_USE = 0x00080000_u32
+  SSL_OP_NO_TICKET = 0x00004000_u32
+  SSL_OP_CIPHER_SERVER_PREFERENCE = 0x00400000_u32
```

Comment:
Maybe the options should be an enum? `SSL::Context#options` would then return an enum and be nicer to deal with.